packages/terra-table/tests/wdio/single-select-table-spec.js

```diff
@@ -1,73 +1,80 @@
-const viewports = Terra.viewports('medium');
-
-describe('Single-Select Table', () => {
+Terra.describeViewports('Single-Select Table', ['medium'], () => {
```

Comment:
How come the selections changed in these screenshots?

Reply:
The `beforeEach` hook was causing the 2nd and 3rd rows to not get selected like the test names suggested they were. Not sure why the focus glow became yellow.

Reply:
The yellow focus glow is related to the Chrome version we are testing